### PR TITLE
RC Edit and view: updated to fit current UI

### PR DESCRIPTION
**Pull request by Sawdustie**
DOC-1599

- Updated configuration tab screenshot

- Changed protocol to type and delete second instance in table

- Updated scalability section to include options for fixed/free

- Changed Redis password to Default user password

- Updated delete database screenshot and language

### CODE REVIEW

File: content/rc/databases/view-edit-database.md [hunk 1]
```diff
@@ -135,6 +134,10 @@ Here, you can:
 
     {{<image filename="images/rc/database-delete-confirm-dialog.png" alt="The Delete database confirmation dialog confirms your decision to delete a database." >}}{{< /image >}}
 
+    When you choose this action for the last database in a subscription, you're shown a different prompt. You're asked to keep the database, delete the database, or delete both the database and the subscription.
```
Comment (rrelledge) [inline content/rc/databases/view-edit-database.md:137]:
```suggestion
    If you only have one database in your subscription, you can delete both the database and the subscription from the **Delete database** confirmation dialog:
        
    - **Delete both** deletes both the database and the subscription.
    
    - **Delete database** deletes the database but keeps the subscription.
```

File: content/rc/databases/view-edit-database.md [hunk 2]
```diff
@@ -135,6 +134,10 @@ Here, you can:
...
+
+    {{<image filename="images/rc/database-delete-last-dialog.png" alt="A different delete database confirmation dialog asks you to consider deleting the subscruption as well.">}}{{< /image >}}
```
Comment (rrelledge) [inline content/rc/databases/view-edit-database.md:139]:
```suggestion
    {{<image filename="images/rc/database-delete-last-dialog.png" alt="A different delete database confirmation dialog asks you to consider deleting the subscription as well.">}}{{< /image >}}
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Looks good! I just left a few suggestions.

**Review by cmilesb — APPROVED** [review]
Looks good to me with Rachel's suggestions.

**Review by rrelledge — APPROVED** [review]

**Pull request merged**